magento/pwa-studio
# [Fix bug]: Submit Create Account form have not trimmed prefix/suffix space for all fields

## Description

Currently, create account with situation fill customer information include prefix/suffix spaces then push to server without any space of those.

## Related Issue





Closes #2748.

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Go to Create Account form.
2. Fill date include some spaces at the prefix/suffix any field: First name, Last name, ...
3. Click Create an Account.
4. Verify the system still create account with data filled with trimmed data.
5. Go to Edit Account Information to verify all fields user information don't have any space at prefix/suffix

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
